.aptible.yml

```diff
@@ -0,0 +1,2 @@
+before_release:
+  - bundle exec rake db:migrate
```

Comment:
GitHub thinks there's a missing newline at the end of this file; see the ⊖

https://stackoverflow.com/a/37598990 is an explanation of how to configure RubyMine to avoid this problem.